visidata/clipboard.py

```diff
@@ -36,6 +37,9 @@ def setslice(L, a, b, M):
 Sheet.addCommand('zY', 'syscopy-cell', 'copyToClipboard(cursorDisplay)')
 Sheet.addCommand('gzY', 'syscopy-cells', 'copyToClipboard("\\n".join(vd.sheet.cursorCol.getDisplayValue(r) for r in selectedRows))')
 
+Sheet.addCommand('CTRL-BUTTON1_PRESSED', 'paste-mouse',
```

Comment:
`CTRL-BUTTON1` might be okay for a default binding.  I think the longname should be `syspaste-mouse` though, for symmetry with `syscopy` since the data is coming from the system clipboard.

Reply:
that makes sense - I'll change the name.  what about middle click (kinda like one of the possible pastes on linux systems)?